common/processors/physic_processor.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from common.engine.processor import Processor
+from common.engine.resource_manager import ResourceManager
+from common.resources.physic_resource import PhysicResource
+from common.resources.time_providers.real_time_provider import RealTimeProvider
+
+logger = logging.getLogger(__name__)
+
+
+class PhysicProcessor(Processor):
+    def __init__(self):
+        super().__init__()
+        self.max_chrono_value = 1 / 60
+        self.current_chrono_value = 0
+
+    def process(self, r: ResourceManager) -> None:
+        world: PhysicResource = r.get_resource(PhysicResource)
+        time_provider: RealTimeProvider = r.get_resource(RealTimeProvider)
```

Comment:
`def get_resource(self, resource: type[R]) -> R`
```suggestion
        time_provider = r.get_resource(RealTimeProvider)
```